### ISSUE TITLE
A11y_AzureDataStudioSQL_PublishProjects_Generate Scripts(Terminal)_Zoom: Data is not properly visible under Task(terminal)at 200% zoom

### ISSUE THREAD

**Issue by anthonydresser** [anthonydresser/testissues#4885]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** -A11yWCAG2.1;-A11yTCS;-A11ySev2;-SH_AzureDataStudioSQL_Feb2021;-SQL Azure Data Studio;-Win10;-FTP;-DesktopApp;-WCAG1.4.4;-AINotInScope;-Zoom;

--- Prerequisite: 

- Set the text size to 200% in "Ease of access settings"

--- Environment Details:
Application Name: Azure Data Studio
Version: 1.26.1
Windows Version: Win10

--- Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Tab till SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an right click on any created project.
5. Tab till "Publish Project" and hit enter
6. Hit on Generate script and navigate to task
7. Check if data is properly visible under task

--- Actual:
Data is not properly visible under Task(terminal)at 200% zoom

--- Expected:
All the data should be properly visible at 200% zoom.

--- User Impact:
Low Vision users will face difficulty in getting the data.
